bootstrap/cmd/bootstrap/app/kfctlClient.go

```diff
@@ -159,27 +167,29 @@ func (c *KfctlClient) CreateDeployment(ctx context.Context, req kfdefs.KfDef) (*
 	return response, nil
 }
 
-func (c *KfctlClient) GetLatestKfdef(req kfdefs.KfDef) (*kfdefs.KfDef, error) {
-	resp, err := c.getEndpoint(context.Background(), req)
-	if err != nil {
-		return nil, err
-	}
-	response, ok := resp.(*kfdefs.KfDef)
-
-	if ok {
-		return response, nil
-	}
-
-	log.Info("Response is not type *KfDef")
-	resErr, ok := resp.(*httpError)
-
-	if ok {
-		return nil, resErr
-	}
-
-	log.Info("Response is not type *httpError")
-
-	pRes, _ := Pformat(resp)
-	log.Errorf("Received unexpected response; %v", pRes)
-	return nil, fmt.Errorf("Received unexpected response; %v", pRes)
+func (c *KfctlClient) GetLatestStatus(req KfctlRequest) (*kfconfig.Status, error) {
```

Comment:
Return type should full KfConfig not just status
Add a TODO about why its commented out.